angular.json

```
@@ -15,15 +15,15 @@
           "options": {
             "allowedCommonJsDependencies": [
               "class-transformer",
-              "rx-polling",
+              "rx-polling-hmcts",
               "util",
               "@edium/fsm",
               "applicationinsights-js",
               "crypto-js",
               "jwt-decode"
             ],
             "aot": true,
-            "outputPath": "dist/rpa-pui-manager",
+            "outputPath": "dist/rpx-xui-manage-organisations",
             "index": "src/index.html",
             "main": "src/main.ts",
             "polyfills": "src/polyfills.ts",
```

Could you enable sourceMap to the non-prod options so it's possible to debug locally?
```suggestion
            "polyfills": "src/polyfills.ts",
            "sourceMap": true
```